src/App.css

```diff
@@ -228,14 +230,45 @@ em {
   font-weight: bold;
   text-decoration: none;
   max-width: max-content;
+  align-self: center;
   box-shadow: 0px 0px 8px 2px rgba(30, 30, 30, 0.1);
-
 }
 .link-to-slides:hover {
   background-color: #1EBDF4;
   color: white;
 }
 
+.filters-intro {
+  justify-content: space-between;
+}
+#fullscreen {
+  font-family: 'Source Sans Pro', sans-serif;
+  align-self: left;
+  border: 2px #1EBDF4 solid;
+  font-size: 16px;
+  color: black;
+  padding: 0.5em;
+  background-color: #EBF9FE;
+}
+#fullscreen:hover {
+  opacity: 0.7;
+}
+.cnnHolder {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.filterGIF-container {
+  width:100%;
+  display: flex;
+  align-items: center;
+}
+
+#filterGIF {
+  align-self: center;
+  width: 250px;
+  height: auto;
+}
```

Comment:
Not merge-blocking, but there's a ton of overlap between these three classes (and other ones in the stylesheet); eventually, it may be good to consolidate the classes. I mentioned this in #20, there's no rush!